frontend/components/Goals/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import image from './images/Squiggles Monochromatic.png';
+import ContentLoader from 'react-content-loader';
```

Comment:
Wow, what a great job here, but here the component will be a dump component that has only one specific mission whish is displaying the content. It will not be responsible for fetching any data, so the use of content loader here is useless